jbpm-flow/src/main/java/org/jbpm/workflow/core/node/WorkItemNode.java

```diff
@@ -92,6 +92,9 @@ public Map<String, String> getInMappings() {
     }
 
     public void addInAssociation(DataAssociation dataAssociation) {
+        if (dataAssociation.getTarget().equals("Description")) {
```

Comment:
I was wondering if we want to do the replacement for any other property different than description?